ZuluContent/Engines/Harvest/Lumberjacking.cs

```diff
@@ -59,6 +61,9 @@ private Lumberjacking()
             // Ten logs per harvest action
             lumber.ConsumedPerHarvest = skillValue => (int) (skillValue / 15) + 1;
 
+            // Maximum chance to roll for colored veins
+            lumber.MaxChance = 100;
```

Comment:
You can simplify the repetitive object assignments using the object initializer syntax
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/object-and-collection-initializers#object-initializers